### ISSUE TITLE
LinkedArt, FRBR et CRMba ? 

### ISSUE THREAD

**Issue by carolinecorbieres** [publicarchi/objectModel#2]
Après relecture de l'article de A. Guillem, G. Bruseker et P. Ronzino, je me demande si LinkedArt permettrait l'intégration de FRBR et CRMba ? 
Les auteurs insistent notamment sur l'utilité du modèle FRBR pour décrire la conception intellectuelle des œuvres architecturales.

**Comment by emchateau** [comment]
C’est une des questions que je dois poser aux collègues, de savoir quel est l’état de la réflexion au sein de Linked Art sur FRBR. Ce que je sais c’est que pour le moment cela n’est pas le cas. Mais c’est évidemment une extension souhaitable à mon avis.
Pour nous, la question sera de savoir si on adopte FRBR ou la proposition de l’article. 
Est-ce que cela répond à votre question ? Voyez-vous quelque chose qui s’opposerait à formaliser les propositions de FRBR et CRMba en suivant les principes de conception adoptés par LinkedArt ?

**Comment by carolinecorbieres** [comment]
Oui, l'intégration de FRBR dans LinkedArt pourrait notamment permettre de traiter les phénomènes éditoriaux ainsi que la création intellectuelle des œuvres. 
Je me penche vendredi sur les principes de conception de LinkedArt pour répondre à votre dernière question.

**Comment by carolinecorbieres** [comment]
Il me semble qu'il serait tout à fait possible d'intégrer FRBR et CRMba dans LinkedArt puisque ces deux modèles sont des extensions de CIDOC CRM, ontologie sur laquelle LinkedArt se base. Rien, sur le site, ne précise que nous ne pourrions pas utiliser ces extensions. 

Je me demande toutefois si CRMba nous serait réellement utile, puisqu'il semble plus être utilisé pour décrire finement des monuments (notamment les phénomènes de modifications architecturales, reconstructions...). D'autant plus que les auteurs de l'article font uniquement référence à la classe B1 Built Work. 
D'autre part, ils proposent une nouvelle série de classes AR (permettant de décrire les diverses étapes de la conception des objets architecturaux), or celles-ci ne figurent dans aucunes des extensions CIDOC. 
Une des solutions pourrait être de partir de FRBR (comme suggéré dans l'article) et de typer les classes avec un thésaurus à l'image de LinkedArt ?
